docs/source/how-to-contribute.rst

```diff
@@ -5,7 +5,7 @@
 How to Contribute
 ==================
 
-We would appreciate seeing you contributing to the `PyExperimenter`. If you have any new idea or have found a bug, please first make sure to check the existing `GitHub Issues <github_py_experimenter_issues_>`_. In case someone else have had a similar idea or has found the same bug, you could give further feedback on how to improve. Otherwise, please :ref:`create an according issue <contribute_create_issue>`. You are planning to work on this issue yourself, or you have found another issue you would like to work on? Great! Directly create a comment stating what exactly you are planning to do and :ref:`setup the development workflow <contribute_fork_project>`.
+We would appreciate seeing you contributing to the ``PyExperimenter``. If you have any new idea or have found a bug, please first make sure to check the existing `GitHub Issues <github_py_experimenter_issues_>`_. In case someone else have had a similar idea or has found the same bug, you could give further feedback on how to improve. Otherwise, please :ref:`create an according issue <contribute_create_issue>`. You are planning to work on this issue yourself, or you have found another issue you would like to work on? Great! Directly create a comment stating what exactly you are planning to do and :ref:`setup the development workflow <contribute_fork_project>`.
```

Comment:
contributing to ``PyExperimenter`` or contributing to the ``PyExperimenter``package